src/Juvix/Compiler/Concrete/Translation/FromSource.hs

```diff
@@ -491,7 +491,7 @@ lambda = do
 -- Data type construction declaration
 -------------------------------------------------------------------------------
 
-inductiveDef :: Members '[InfoTableBuilder, JudocStash, NameIdGen] r => Maybe (WithLoc BuiltinInductive) -> ParsecS r (InductiveDef 'Parsed)
+inductiveDef :: (Members '[InfoTableBuilder, JudocStash, NameIdGen] r) => Maybe (WithLoc BuiltinInductive) -> ParsecS r (InductiveDef 'Parsed)
```

Comment:
Why do these brackets suddenly appear? Can't we have a separate PR to adapt to the new ormolu version?

Reply:
Hum, I don't know. I run `make check`, which calls `make format`.

Reply:
I run `make format` with ormolu version 0.5.2 and it doesn't add these braces, but also doesn't remove them. Strange.

Reply:
Strange, also because the CI didn't complain about the changes.

Reply:
I have the same behaviour as Lukasz. We should try to find a solution for this. Otherwise Jonathan's prs will often include unrelated formatting changes